<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CurrentWeatherWidget</class>
 <widget class="QWidget" name="CurrentWeatherWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ÂÆûÊó∂Â§©Ê∞î</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#CurrentWeatherWidget {
    background-color: #f5f7fa;
}

QFrame#mainCard {
    background-color: white;
    border-radius: 12px;
    border: 1px solid #e4e7ed;
}

QFrame#detailCard {
    background-color: white;
    border-radius: 8px;
    border: 1px solid #e4e7ed;
}

QLabel#cityNameLabel {
    font-size: 24px;
    font-weight: bold;
    color: #303133;
}

QLabel#temperatureLabel {
    font-size: 72px;
    font-weight: bold;
    color: #409EFF;
}

QLabel#weatherDescLabel {
    font-size: 20px;
    color: #606266;
}

QLabel#updateTimeLabel {
    font-size: 12px;
    color: #909399;
}

QLabel.detailTitle {
    font-size: 12px;
    color: #909399;
}

QLabel.detailValue {
    font-size: 16px;
    font-weight: bold;
    color: #303133;
}

QLabel#aqiLabel {
    font-size: 14px;
    font-weight: bold;
    padding: 4px 12px;
    border-radius: 4px;
}

QPushButton#refreshBtn {
    background-color: #409EFF;
    color: white;
    border: none;
    border-radius: 6px;
    padding: 8px 16px;
    font-size: 14px;
}

QPushButton#refreshBtn:hover {
    background-color: #66b1ff;
}

QPushButton#refreshBtn:pressed {
    background-color: #3a8ee6;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>16</number>
   </property>
   <property name="leftMargin">
    <number>20</number>
   </property>
   <property name="topMargin">
    <number>20</number>
   </property>
   <property name="rightMargin">
    <number>20</number>
   </property>
   <property name="bottomMargin">
    <number>20</number>
   </property>
   <item>
    <widget class="QFrame" name="mainCard">
     <property name="frameShape">
      <enum>QFrame::Shape::StyledPanel</enum>
     </property>
     <layout class="QVBoxLayout" name="mainCardLayout">
      <property name="spacing">
       <number>12</number>
      </property>
      <property name="leftMargin">
       <number>24</number>
      </property>
      <property name="topMargin">
       <number>24</number>
      </property>
      <property name="rightMargin">
       <number>24</number>
      </property>
      <property name="bottomMargin">
       <number>24</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="headerLayout">
        <item>
         <widget class="QLabel" name="cityNameLabel">
          <property name="text">
           <string>ÈÄâÊã©ÂüéÂ∏Ç</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="headerSpacer">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="refreshBtn">
          <property name="text">
           <string>Âà∑Êñ∞</string>
          </property>
          <property name="icon">
           <iconset theme="view-refresh"/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="mainInfoLayout">
        <property name="spacing">
         <number>24</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="tempLayout">
          <item>
           <widget class="QLabel" name="temperatureLabel">
            <property name="text">
             <string>--¬∞</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="feelsLikeLabel">
            <property name="text">
             <string>‰ΩìÊÑüÊ∏©Â∫¶ --¬∞</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignCenter</set>
            </property>
            <property name="styleSheet">
             <string>color: #909399; font-size: 14px;</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="weatherLayout">
          <item>
           <widget class="QLabel" name="weatherIconLabel">
            <property name="minimumSize">
             <size>
              <width>80</width>
              <height>80</height>
             </size>
            </property>
            <property name="text">
             <string>‚òÄ</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignCenter</set>
            </property>
            <property name="styleSheet">
             <string>font-size: 64px;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="weatherDescLabel">
            <property name="text">
             <string>--</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="mainInfoSpacer">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QVBoxLayout" name="aqiLayout">
          <item>
           <widget class="QLabel" name="aqiTitleLabel">
            <property name="text">
             <string>Á©∫Ê∞îË¥®Èáè</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignCenter</set>
            </property>
            <property name="styleSheet">
             <string>color: #909399; font-size: 12px;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="aqiLabel">
            <property name="text">
             <string>--</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="aqiValueLabel">
            <property name="text">
             <string>AQI --</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignCenter</set>
            </property>
            <property name="styleSheet">
             <string>color: #606266; font-size: 14px;</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="updateTimeLabel">
        <property name="text">
         <string>Êõ¥Êñ∞Êó∂Èó¥: --</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignVCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="detailsLayout">
     <property name="spacing">
      <number>12</number>
     </property>
     <item>
      <widget class="QFrame" name="detailCard1">
       <property name="frameShape">
        <enum>QFrame::Shape::StyledPanel</enum>
       </property>
       <layout class="QVBoxLayout" name="detail1Layout">
        <item>
         <widget class="QLabel" name="humidityTitle">
          <property name="text">
           <string>ÊπøÂ∫¶</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignCenter</set>
          </property>
          <property name="styleSheet">
           <string>color: #909399; font-size: 12px;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="humidityLabel">
          <property name="text">
           <string>--%</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignCenter</set>
          </property>
          <property name="styleSheet">
           <string>font-size: 18px; font-weight: bold; color: #303133;</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="detailCard2">
       <property name="frameShape">
        <enum>QFrame::Shape::StyledPanel</enum>
       </property>
       <layout class="QVBoxLayout" name="detail2Layout">
        <item>
         <widget class="QLabel" name="windTitle">
          <property name="text">
           <string>È£éÈÄü</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignCenter</set>
          </property>
          <property name="styleSheet">
           <string>color: #909399; font-size: 12px;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="windLabel">
          <property name="text">
           <string>-- km/h</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignCenter</set>
          </property>
          <property name="styleSheet">
           <string>font-size: 18px; font-weight: bold; color: #303133;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="windDirLabel">
          <property name="text">
           <string>--</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignCenter</set>
          </property>
          <property name="styleSheet">
           <string>color: #606266; font-size: 12px;</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="detailCard3">
       <property name="frameShape">
        <enum>QFrame::Shape::StyledPanel</enum>
       </property>
       <layout class="QVBoxLayout" name="detail3Layout">
        <item>
         <widget class="QLabel" name="pressureTitle">
          <property name="text">
           <string>Ê∞îÂéã</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignCenter</set>
          </property>
          <property name="styleSheet">
           <string>color: #909399; font-size: 12px;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="pressureLabel">
          <property name="text">
           <string>-- hPa</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignCenter</set>
          </property>
          <property name="styleSheet">
           <string>font-size: 18px; font-weight: bold; color: #303133;</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="detailCard4">
       <property name="frameShape">
        <enum>QFrame::Shape::StyledPanel</enum>
       </property>
       <layout class="QVBoxLayout" name="detail4Layout">
        <item>
         <widget class="QLabel" name="visibilityTitle">
          <property name="text">
           <string>ËÉΩËßÅÂ∫¶</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignCenter</set>
          </property>
          <property name="styleSheet">
           <string>color: #909399; font-size: 12px;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="visibilityLabel">
          <property name="text">
           <string>-- km</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignCenter</set>
          </property>
          <property name="styleSheet">
           <string>font-size: 18px; font-weight: bold; color: #303133;</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="sunLayout">
     <property name="spacing">
      <number>12</number>
     </property>
     <item>
      <widget class="QFrame" name="sunriseCard">
       <property name="frameShape">
        <enum>QFrame::Shape::StyledPanel</enum>
       </property>
       <layout class="QHBoxLayout" name="sunriseLayout">
        <item>
         <widget class="QLabel" name="sunriseIcon">
          <property name="text">
           <string>üåÖ</string>
          </property>
          <property name="styleSheet">
           <string>font-size: 32px;</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="sunriseInfoLayout">
          <item>
           <widget class="QLabel" name="sunriseTitle">
            <property name="text">
             <string>Êó•Âá∫</string>
            </property>
            <property name="styleSheet">
             <string>color: #909399; font-size: 12px;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="sunriseLabel">
            <property name="text">
             <string>--:--</string>
            </property>
            <property name="styleSheet">
             <string>font-size: 18px; font-weight: bold; color: #303133;</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="sunsetCard">
       <property name="frameShape">
        <enum>QFrame::Shape::StyledPanel</enum>
       </property>
       <layout class="QHBoxLayout" name="sunsetLayout">
        <item>
         <widget class="QLabel" name="sunsetIcon">
          <property name="text">
           <string>üåá</string>
          </property>
          <property name="styleSheet">
           <string>font-size: 32px;</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="sunsetInfoLayout">
          <item>
           <widget class="QLabel" name="sunsetTitle">
            <property name="text">
             <string>Êó•ËêΩ</string>
            </property>
            <property name="styleSheet">
             <string>color: #909399; font-size: 12px;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="sunsetLabel">
            <property name="text">
             <string>--:--</string>
            </property>
            <property name="styleSheet">
             <string>font-size: 18px; font-weight: bold; color: #303133;</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="bottomSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
